dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/metadata/ConfigurableMetadataServiceExporter.java

```diff
@@ -65,13 +66,12 @@ public synchronized ConfigurableMetadataServiceExporter export() {
                     .getInternalServiceExecutor();
             this.serviceConfig = InternalServiceConfigBuilder.<MetadataService>newBuilder(applicationModel)
                     .interfaceClass(MetadataService.class)
-                    .protocol(getApplicationConfig().getMetadataServiceProtocol(), METADATA_SERVICE_PROTOCOL_KEY)
+                    .protocol(TRIPLE, METADATA_SERVICE_PROTOCOL_KEY)
```

Comment:
so `application.metadataServiceProtocol ` will not work in the future

Reply:
If we always use tri protocol, how can we compatiable with old version client

Reply:
Yes. I will do some refactor later to make sure it can compatible with older version.